packages/frontend/src/features/scheduler/components/SchedulersListItem.tsx

```diff
@@ -31,47 +27,53 @@ const SchedulersListItem: FC<SchedulersListItemProps> = ({
     onDelete,
 }) => {
     return (
-        <SchedulerContainer>
-            <SchedulerDetailsContainer
-                className={Classes.TEXT_OVERFLOW_ELLIPSIS}
-            >
-                <SchedulerName>{scheduler.name}</SchedulerName>
-                <Menu withArrow withinPortal width={100}>
-                    <Menu.Target>
-                        <MantineIcon icon={IconDots} />
-                    </Menu.Target>
+        <Paper p="sm" mb="xs" withBorder sx={{ overflow: 'hidden' }}>
+            <Group noWrap position="apart">
+                <Stack spacing="xs" w={475}>
+                    <Text fw={600} truncate>
+                        {scheduler.name}
+                    </Text>
+                    <Group>
+                        <Text color="gray" size={12}>
+                            {getHumanReadableCronExpression(scheduler.cron)}
+                        </Text>
 
-                    <Menu.Dropdown>
-                        <Menu.Item
-                            icon={<MantineIcon icon={IconPencil} />}
-                            onClick={() => onEdit(scheduler.schedulerUuid)}
+                        {/* TODO: This icon should use Mantine icon,
+                            but MantineIcon doesn't support filled icons atm.
+                            Util we fix that, this style is imperfect
+                        */}
+                        <Box
+                            sx={(theme) => ({
+                                color: theme.colors.gray[4],
+                                marginTop: '-6px',
+                            })}
                         >
-                            Edit
-                        </Menu.Item>
-                        <Divider />
+                            <IconCircleFilled style={{ width: 5, height: 5 }} />
+                        </Box>
```

Comment:
Should we use CSS to create the circle instead of an icon?
eg
```
<Box
    sx={(theme) => ({
        width: 5,
        height: 5,
        background-color: theme.colors.gray[4],
        border-radius: '50%',
    })}
/>
```

Reply:
That could work here, but the icon is probably the right idea for re-usability. I'll see if I can fix that wrapper class -- we're going to want that elsewhere.